API.md

```diff
@@ -422,6 +422,32 @@ Example Response:
 }
 ```
 
+`POST /api/users` - Creates a user account, this is called on sign-up.  
+Request returns a javascript web token.
+
+Example Request:
+```javascript
+// POST /api/users/
+
+{
+  name: "John Doe",
+  email: "John@Doe.com",
+  password: "password",
+  isInstructor: false
+}
+```
+
+Example Response:
+```javascript
+
+{
+  $promise: {Promise Object},
+  $resolved: true,
```

Comment:
:penguin: i probably wouldn't have included it and it actually should just be removed completely from the API haha